jr-annotation-support/src/test/java/com/fasterxml/jackson/jr/annotationsupport/ASTestBase.java

```diff
@@ -2,17 +2,15 @@
 
 import java.util.Arrays;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonValue;
 import com.fasterxml.jackson.jr.ob.JSON;
 
 import junit.framework.TestCase;
 
 public abstract class ASTestBase extends TestCase
 {
-    protected enum ABC { A, B, C; }
-
-    protected enum ABCRename { @JsonProperty("A1") A, @JsonProperty("B1") B, C; }
-
```

Comment:
Only used in the subclass. I originally added to this base class to follow the original pattern. Have tidied up a little.